jastrow.py

```diff
@@ -25,12 +49,36 @@ def eidist(configs, coords):
     return d
     
 
+def eidist(configs, coords, nup, ndown):
+    """returns a list of electron-ion distances"""
+    ne=configs.shape[1]
+    ni=len(coords)
+    d1=np.zeros((configs.shape[0],nup,ni,3))   # up case
+    d2=np.zeros((configs.shape[0],ndown,ni,3)) # down case
+
+    # First electrons are spin up by convenction
+    for i in range(ne):
+        for j in range(ni):
+            if(i<nup): #Spin up case
```

Comment:
Same applies here, loop separately instead of if statements.

Reply:
I also changed this to loop over individual populations.